src/project/projectSlice.ts

```diff
@@ -16,6 +16,8 @@
  */
 
 import { createSlice } from '@reduxjs/toolkit';
+import { User } from 'terraso-client-shared/account/accountSlice';
+import { Membership } from 'terraso-client-shared/memberships/membershipsSlice';
```

Comment:
question: since we are trying to keep the fact that `Project`, `Landscape`, `Group` share a membership table a backend implementation detail, should the project slice define its own `ProjectMembership` type?

Reply:
Yes, that seems like a good idea to me. Do you think this should be in the client-shared repo, or here?